main/src/cgeo/geocaching/network/StatusUpdater.java

```diff
@@ -81,7 +79,7 @@ static Status defaultStatus(final Status upToDate) {
             } catch (NumberFormatException e) {
                 // skip version check if no parseable number returned
             }
-            return Build.VERSION.SDK_INT >= 30 ? ANDROID11NOTICE_STATUS : Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP ? CLOSEOUT_STATUS : NO_STATUS;
+            return Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP ? CLOSEOUT_STATUS : NO_STATUS;
```

Comment:
SDK_INT is never < 21. Please change to 
`  return NO_STATUS;`
see https://ci.cgeo.org/view/Continous%20integration/job/cgeo%20continuous%20integration/3047/androidLintResult/type.398387310/

or add `@SuppressLint(" ObsoleteSdkInt")` if it should stay as template.

Reply:
The latter, so I'll add a `@SuppressLint`